src/context_store/storage/sqlite.py

```
@@ -758,34 +758,70 @@ async def vector_search(
-                        SELECT m.*, me.embedding, (-bm25(memories_fts)) AS score
-                        FROM memories_fts f
-                        JOIN memories m ON m.rowid = f.rowid
-                        LEFT JOIN memory_embeddings me ON me.memory_id = m.id
-                        WHERE memories_fts MATCH ?
-                          AND m.archived_at IS NULL
-                          AND m.project = ?
-                        ORDER BY score DESC
-                        LIMIT ?
-                    """
-                    params_kw: tuple[Any, ...] = (query, project, top_k)
+                # 空クエリ / 空白のみは「すべてマッチ」として扱う（Postgres との互換性）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)